internal/gatewayapi/backendtrafficpolicy.go

```diff
@@ -423,15 +423,29 @@ func (t *Translator) buildLoadBalancer(policy *egv1a1.BackendTrafficPolicy) *ir.
 		}
 	case egv1a1.LeastRequestLoadBalancerType:
 		lb = &ir.LoadBalancer{
-			LeastRequest: &ir.LeastRequest{},
+			LeastRequest: &ir.LeastRequest{
+				SlowStart: &ir.SlowStart{},
```

Comment:
instead of setting it here , lets set it under `policy.Spec.LoadBalancer.SlowStart != nil`